src/main/java/ru/ayaz/User.java

```diff
@@ -0,0 +1,54 @@
+package ru.ayaz;
+
+import java.io.PrintWriter;
+import java.io.Writer;
+import java.security.Principal;
+
+/**
+ * Created by ayaz on 11/5/16.
+ */
+public class User {
+    private String nickName;
+    private long id;
```

Comment:
Do you mean "id" field? I am going to use "id" as key to my TreeMap for future developing blockList and banList of simpleUser and AdminUser respectively (will change class User to interface).